packages/react-ds/src/components/Pagination/Pagination.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import {
+  Meta,
+  Story,
+  Description,
+  Preview,
+  Props,
+} from '@storybook/addon-docs/blocks'
+
+import Pagination from './Pagination'
+import { withA11y } from '@storybook/addon-a11y'
+import { withKnobs } from '@storybook/addon-knobs'
+import Docs from '@venice/core/docs/Pagination.md'
+import { knobPaginationIsMobile, knobPaginationIsLoading } from '@venice/core/storybook/knobs'
+
+
+<Meta title="Components|Pagination" component={Pagination} />
+
+<Description>{Docs}</Description>
+
+### Default
+
+<Preview>
+  <Story name="Default" decorators={[withKnobs,withA11y]}>
+    <div>
+      <Pagination count={7} page={1} onChangePage={() => {}} isLoading={knobPaginationIsLoading()} />
```

Comment:
fix `onChangePage`on there

Reply:
omg, thanks. I changed the prop name and i forgot to change in .mdx file hehehe

Reply:
made on: 2db38a9